Uplift/Controllers/OnboardingGymsViewController.swift

```diff
@@ -51,6 +38,8 @@ class OnboardingGymsViewController: UIViewController, UITableViewDelegate, UITab
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .fitnessWhite
+
+        let buttonBorderSize: CGFloat = 2
```

Comment:
This used to be outside under // Display Info but while the other variables were either made private or put under setUpConstraints you put this one variable inside viewDidLoad. How come?

Reply:
buttonBorderSize is only used to make the nextButton and backButton, which is only done in viewDidLoad(), hence the move to that func only and not as a class field